types/src/v17/network/mod.rs

```diff
@@ -263,4 +263,13 @@ pub struct ListBanned(pub Vec<Banned>);
 
 /// An item from the list returned by the JSON-RPC method `listbanned`
 #[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
-pub struct Banned(String); // FIXME: The docs are empty so I don't know what shape this is.
+pub struct Banned {
```

Comment:
Can we have some docs saying that the shape of this type was pulled from Core source code because the method does not show up in the Bitcoin Core RPC CLI docs.